source/documentation/deploying_apps/production_checklist.md

```diff
@@ -1,9 +1,15 @@
 ## Production checklist
 
-Before deploying an app for production use, check the following:
+Before you deploy an app to a production environment, you must:
 
-1. If your app uses a [backing service](/deploying_services/#deploy-a-backing-or-routing-service), make sure that you have selected the high-availability plan for the service instance.
-1. You have [a domain name](https://www.gov.uk/service-manual/technology/get-a-domain-name) for your service.
-1. You are running more than one instance of the app to ensure availability. Use ``cf scale APPNAME -i INSTANCES`` or amend the manifest file to add more running instances; see [Scaling](/managing_apps.html#scaling) for more information.
-1. You are prepared to use a [blue-green deployment process](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) for when the app needs to be updated or restarted (this avoids problems due to a known issue with the PaaS that can generate transient 404 errors).
-1. You have configured a [health check](https://docs.cloudfoundry.org/devguide/deploy-apps/healthchecks.html) by setting `health-check-type` to `http` and setting `health-check-http-endpoint` to an appropriate endpoint for your app.
+- [get a domain from GOV.UK](https://www.gov.uk/service-manual/technology/get-a-domain-name) for your service and set the [DNS records required by your cdn-route service](https://docs.cloud.service.gov.uk/deploying_services/use_a_custom_domain/#set-up-a-cdn-route-service-with-one-or-more-custom-domains) on the GOV.UK PaaS
```

Comment:
```suggestion
- get a domain for your service and set the [DNS records required by your cdn-route service](/deploying_services/use_a_custom_domain/#set-up-a-cdn-route-service-with-one-or-more-custom-domains)
  - for central government services, which use the GOV.UK domain, refer to the [service manual guidance](https://www.gov.uk/service-manual/technology/get-a-domain-name)
  - if using your own CDN, refer to [the relevant documentation](/deploying_services/configure_cdn/#configure-your-custom-domain-in-cloud-foundry)
```

Reply:
thanks Toby, I've added your suggestion in, I've just amended it slightly to avoid nested bullets due to style considerations